cromshell

```diff
@@ -207,6 +210,20 @@ function usage()
   echo -e "  Update cromwell server:"
   echo -e "   update-server          Change which cromwell server jobs will be submitted to."
   echo -e ""
+  echo -e "  Get cost for a workflow"
+  echo -e "   cost [workflow-id] [[workflow-id]...]                  Get the cost for a workflow."
+  echo -e "                                                          Only works for workflows that completed"
+  echo -e "                                                          more than 8 hours ago on GCS."
+  echo -e "                                                          Requires the 'gcp_bq_cost_table.config'"
+  echo -e "                                                          configuration file to exist and contain"
```

Comment:
Is there somewhere we could link to that would explain how to find that table?  Is it automatically generated in every google project?

Reply:
I have no idea how to find it.  I saw it in a document that was written based on a post by @kcibul … There is probably a way to do it, but I don't know what it is.

Reply:
Huh, mysterious.